tests/integration/index/query_with_relation_filter_test.go

```diff
@@ -265,7 +265,7 @@ func TestQueryWithIndexOnOneToOnePrimaryRelation_IfFilterOnIndexedFieldOfRelatio
 			},
 			testUtils.Request{
 				Request:  makeExplainQuery(req2),
-				Asserter: testUtils.NewExplainAsserter().WithFieldFetches(15).WithIndexFetches(3),
+				Asserter: testUtils.NewExplainAsserter().WithFieldFetches(33).WithIndexFetches(3),
```

Comment:
question: Did this change because of the production code change, or because of the Playstation 5 year change?

Reply:
it's because the production code changed. The planner now doesn't stop after finding first matching related secondary doc and continues until all sec. docs are exhausted.